includes/publisher/js/src/App.jsx

```diff
@@ -1,10 +1,35 @@
 import React from "react";
+import ReactGA from "react-ga4";
 import Fields from "./components/Fields";
 import { sprintf, __ } from "@wordpress/i18n";
 
+const GA_ID = "G-S056CLLZ34";
+
+/**
+ * React-GA4 Library and usage
+ * {@link https://github.com/PriceRunner/react-ga4#readme}
+ *
+ * Page send
+ * ReactGA.send("pageview"); OR
+ * ReactGA.send({ hitType: "pageview", page: "/my-path" });
+ *
+ * Send custom event
+ * ReactGA.event({
+ *	category: "your category",
+ *	action: "your action",
+ *	label: "your label", // optional
+ *	value: 99, // optional, must be a number
+ *	nonInteraction: true, // optional, true/fals
+ *	transport: "xhr", // optional, beacon/xhr/image
+ *	});
+ */
+ReactGA.initialize(GA_ID, { gtagOptions: { anonymize_ip: true } });
+
 export default function App({ model, mode }) {
 	const isEditMode = mode === "edit";
 
+	ReactGA.send({ hitType: "pageview", page: "Publisher App Home" });
```

Comment:
What do you think of this? (The publisher app doesn't really have a “Home”, but if the publisher app loads the user is viewing a publisher entry.)

```suggestion
	ReactGA.send({ hitType: "pageview", page: "ACM Publisher Entry" });
```